src/openrct2-ui/windows/ScenarioSelect.cpp

```diff
@@ -82,76 +81,59 @@ enum {
     WIDX_SCENARIOLIST
 };
 
+static constexpr const StringId ScenarioOriginStringIds[] = {
+    STR_SCENARIO_CATEGORY_RCT1,        STR_SCENARIO_CATEGORY_RCT1_AA,    STR_SCENARIO_CATEGORY_RCT1_LL,
+    STR_SCENARIO_CATEGORY_RCT2,        STR_SCENARIO_CATEGORY_RCT2_WW,    STR_SCENARIO_CATEGORY_RCT2_TT,
+    STR_SCENARIO_CATEGORY_UCES,        STR_SCENARIO_CATEGORY_REAL_PARKS, STR_SCENARIO_CATEGORY_EXTRAS_PARKS,
+    STR_SCENARIO_CATEGORY_OTHER_PARKS,
+};
+// clang-format off
```

Comment:
Add blank lines between unrelated parts.
```suggestion

// clang-format off
```